src/concrete/ml/sklearn/base.py

```diff
@@ -1482,6 +1490,54 @@ def __init__(self, n_bits: Union[int, Dict[str, int]] = 8):
 
         BaseEstimator.__init__(self)
 
+    @classmethod
+    def from_sklearn_model(
```

Comment:
this is the new feature

Reply:
That's awesome!

Reply:
In BaseEstimator so all models have access to this feature?

Reply:
tree-based and QNN models both need re-fitting with quantizers, so no it's only a feature for linear models for noow